OpenEdXMobile/res/drawable/bt_google.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <corners android:radius="2dp" />
```

Comment:
The file's name and this dimension should be matched with that of Facebook's button xml i.e. `fb_btn_bg.xml`